### PR TITLE
Update Imperium - Grey Knights.cat

### PR DESCRIPTION
**Pull request by PappyJLoco**
Fixing some typos. Also change dsome of the capitalization on Bolter and Grenades - unsure if that has a knock on impact for lookups?

### CODE REVIEW

**Comment by Mad-Spy** [comment]
- A number of these changes are incorrect ("psychic powers" and "psychic action" never have a capital P, only the "Psychic phase" does.). 
- Changing the capitalisation of a link name does not affect the actual link (you should have seen this when checking your changes?)

File: Imperium - Grey Knights.cat [hunk 1]
```diff
@@ -307,7 +307,7 @@
             <characteristic name="S" typeId="59b1-319e-ec13-d466">+1</characteristic>
             <characteristic name="AP" typeId="75aa-a838-b675-6484">-3</characteristic>
             <characteristic name="D" typeId="ae8a-3137-d65b-4ca7">2</characteristic>
-            <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Each time an attack is made with this weapon, an unmodified wound roll of 6 inflictct D3 mortal wounds on the target in addition to any normal damage</characteristic>
+            <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Each time an attack is made with this weapon, an unmodified wound roll of 6 inflict D3 mortal wounds on the target in addition to any normal damage</characteristic>
```
Comment (Mad-Spy) [inline Imperium - Grey Knights.cat:310]: This should be inflicts?

**Review by Mad-Spy — COMMENTED** [review]
- The majority of these capitalisation changes are incorrect ("psychic powers" and "psychic action" never have a capital P, only the "Psychic phase" does.).
- Changing the capitalisation of a link name makes no difference to the naming in the roster editor (it will always uses the name of the linked item). You should have seen this when you checked your changes? Also, your capitalisation of the second word is incorrect. Weapon profiles and names only have an initial capital, not a capital on every word.

**Comment by PappyJLoco** [comment]
> * The majority of these capitalisation changes are incorrect ("psychic powers" and "psychic action" never have a capital P, only the "Psychic phase" does.).
> * Changing the capitalisation of a link name makes no difference to the naming in the roster editor (it will always uses the name of the linked item). You should have seen this when you checked your changes? Also, your capitalisation of the second word is incorrect. Weapon profiles and names only have an initial capital, not a capital on every word.

Yeah fair enough - How do I go about submitting the new code for review?

**Comment by PappyJLoco** [comment]
I'll close and re-submit.

**Pull request closed without merging**